<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="sqltable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>1050</width>
      <height>590</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView QListView::section 
{

    color: red;
    background-color: rgb(64, 64, 64);
    border: transparent;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
	
 }

QHeaderView 
{
    color: white;
    font: bold 10pt;
    background-color: rgb(108, 108, 108);
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
}

QHeaderView::section 
{
    color: white;
    background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
} 


QTableView QScrollBar:horizontal
{
	background:#FFFFFF;
	padding-top:3px;
	padding-bottom:3px;
	padding-left:20px;
	padding-right:20px;
}

QTableView QScrollBar::handle:horizontal
{
background:#dbdbdb;
border-radius:6px;
min-width:80px;
}
QTableView QScrollBar::handle:horizontal:hover
{
   background:#d0d0d0;
}
QTableView QScrollBar::add-line:horizontal{
    background:url(:/image/scr_right.png) 
}
QTableView QScrollBar::sub-line:horizontal
{
 background:url(:/image/scr_left.png)
}
</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="leavebtn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#leavebtn {	
    color: #fff;
    border: 0px solid rgba(255, 255, 255, 0);
    font-size: 12px;
    font-weight: 700;
    border-radius: 5px;
    background-color: #f4f4f4;
	background-image: url(:/image/logout_red.png);
}

QPushButton#leavebtn:hover {
    background-color: #ffffff;
	background-image: url(:/image/logout_red.png);
}

QPushButton#leavebtn:pressed {
    background: qradialgradient(cx:0.5,
    cy: 0.5,
    fx: 0.5,
    fy: 0.5,
    radius: 1.5,
	stop:0.2 #f4f4f4,
	stop:0.8 #f4f4f4);
	background-image: url(:/image/logout_red.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_item">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#add_item {	
    color: #fff;
    border: 0px solid rgba(255, 255, 255, 0);
    font-size: 12px;
    font-weight: 700;
    border-radius: 5px;
    background-color: #f4f4f4;
	background-image: url(:/image/add.png);
}

QPushButton#add_item:hover {
    background-color: #ffffff;
	background-image: url(:/image/add.png);
}

QPushButton#add_item:pressed {
    background: qradialgradient(cx:0.5,
    cy: 0.5,
    fx: 0.5,
    fy: 0.5,
    radius: 1.5,
	stop:0.2 #f4f4f4,
	stop:0.8 #f4f4f4);
	background-image: url(:/image/add.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label_user
{
	background-image: url(:/image/username.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="current_user">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="del_item">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#del_item {	
    color: #fff;
    border: 0px solid rgba(255, 255, 255, 0);
    font-size: 12px;
    font-weight: 700;
    border-radius: 5px;
    background-color: #f4f4f4;
	background-image: url(:/image/jian.png);
}

QPushButton#del_item:hover {
    background-color: #ffffff;
	background-image: url(:/image/jian.png);
}

QPushButton#del_item:pressed {
    background: qradialgradient(cx:0.5,
    cy: 0.5,
    fx: 0.5,
    fy: 0.5,
    radius: 1.5,
	stop:0.2 #f4f4f4,
	stop:0.8 #f4f4f4);
	background-image: url(:/image/jian.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="seclect_date">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#seclect_date {	
    color: #fff;
    border: 0px solid rgba(255, 255, 255, 0);
    font-size: 12px;
    font-weight: 700;
    border-radius: 5px;
    background-color: #f4f4f4;
	background-image: url(:/image/calen.png);
}

QPushButton#seclect_date:hover {
    background-color: #ffffff;
	background-image: url(:/image/calen.png);
}

QPushButton#seclect_date:pressed {
    background: qradialgradient(cx:0.5,
    cy: 0.5,
    fx: 0.5,
    fy: 0.5,
    radius: 1.5,
	stop:0.2 #f4f4f4,
	stop:0.8 #f4f4f4);
	background-image: url(:/image/calen.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="month">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>30</y>
      <width>137</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid #aab2bd;
    border-radius: 4px;
    font-size: 16px;
    padding: 5px 8px;
}
QComboBox::month {
    background-color: transparent;
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="year">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>138</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QComboBox::year {
	background-color:#ffffff;
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
 
</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>50</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="cur_bill">
       <property name="text">
        <string>当前账单:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cur_bill_box"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
